turbinia/jobs/manager.py

```diff
@@ -108,14 +108,18 @@ def DeregisterJobs(cls, jobs_blacklist=None, jobs_whitelist=None):
     """
     registered_jobs = list(cls.GetJobNames())
     jobs_remove = []
-
     # Create a list of jobs to deregister.
     if jobs_whitelist and jobs_blacklist:
       raise TurbiniaException(
           'jobs_whitelist and jobs_blacklist cannot be specified at the same '
           'time.')
     elif jobs_whitelist:
       jobs_whitelist = [j.lower() for j in jobs_whitelist]
+      for j in jobs_whitelist:
+        if j not in registered_jobs:
+          msg = 'Error whitelisting the worker. Job {} is not supported by Turninia.'.format(
```

Comment:
Small nit: This isn't actually whitelisting the worker, I would probably change this to something like the following to make it more clear: `Error registering Jobs:  Whitelisted Job {0:s} is not found in registered jobs {1!s}.format(j, registered_jobs)`.  Note that I also changed the format string to match the existing code style.

Reply:
Done.